packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/modelparts/ModelMetadata.java

```diff
@@ -43,4 +44,8 @@ public class ModelMetadata {
 	@JsonProperty("card")
 	@TSOptional
 	private Card card;
+
+	@JsonProperty("provenance")
```

Comment:
The `@JsonProperty` here isn't needed as the field seems to match the incoming JSON name... but leaving it isn't wrong either. just a FYI!